File: benchmark/results/java-invocation-script/run-001/b-baseline/output/com/metaengine/demo/billing/services/InvoiceService.java
package com.metaengine.demo.billing.services;

import com.metaengine.demo.billing.aggregates.Invoice;
import java.util.List;

/** InvoiceService for the billing domain. */
public class InvoiceService {

    /** Create a new Invoice. */
    public Invoice create(Invoice input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Invoice by id; may return null. */
    public Invoice findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List invoices up to the given limit. */
    public List<Invoice> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an invoice by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/billing/services/invoice_service.py
"""InvoiceService for the billing domain."""
from typing import Optional

from metaengine_demo.billing.aggregates.invoice import Invoice


class InvoiceService:
    """InvoiceService application service."""

    def create(self, input: Optional[Invoice]) -> Invoice:
        """Create a new invoice."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Invoice]:
        """Find an invoice by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Invoice]:
        """List invoices up to a limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete an invoice by id."""
        raise NotImplementedError("not implemented")
